meet_balls_app/views.py

```diff
@@ -27,3 +31,63 @@ def loginUser(request):
 def logoutUser(request):
     logout(request)
     return render(request, 'meet_balls_app/home.html', {'user': {'is_authenticated': False}})
+
+
+def registerUser(request):
+    if request.method == 'POST':
+        username = request.POST.get('username')
+        password = request.POST.get('password')
+        confirm_password = request.POST.get('confirm_password')
+        birthdate = request.POST.get('birthdate')
+        favorite_ball_game = request.POST.get('favorite_ball_game')
+
+        data = {
+            'username': username,
+            'password': password,
+            'confirm_password': confirm_password,
+            'birthdate': birthdate,
+            'favorite_ball_game': favorite_ball_game,
+        }
+        errors = validate_register_form(data)
+        if errors:
+            for error in errors:
+                messages.error(request, error)
+            return render(request, 'meet_balls_app/register.html',
+                          {'ball_games': BallGame.choices})
+        else:
+            try:
+                player = Player.create(
+                    username, password, birthdate, favorite_ball_game)
+                user = player.user
+                login(request, user)
+                messages.error(request, '')
+                return render(request, 'meet_balls_app/home.html',
```

Comment:
@IdanZimi I think it should be `redirect` rather `render`, and maybe it should be better to redirect the user to the login page, where he should log in, and not directly log him in, when he registrates.

Reply:
Fixed. Now It redirects to the login page after successful registration. @OhadSeg